charts/sourcegraph/templates/grafana/grafana.ConfigMap.yaml

```diff
@@ -15,6 +15,15 @@ data:
         type: Jaeger
         access: proxy
         url: http://{{ default "jaeger-query" .Values.jaeger.query.name }}:16686/-/debug/jaeger
+      - name: pgsql
+        type: postgres
+        url: $PGHOST:$PGPORT
+        user: $PGGRAFANAUSER
+        database: $PGDATABASE
+        secureJsonData:
+          password: $PGGRAFANAPASSWORD
+        jsonData:
+          sslmode: $PGSSLMODE
```

Comment:
Can we make this optional with something like `{{- if .Values.grafana.postgres.enabled }}` wrapping it? 
I'd be worried about this being on by default.

Reply:
Grafana doesn't create this datasource if the env vars are not defined, so the switch to enable it is defining all of these env vars on the Grafana container; it'd be redundant to add another config flag.

Reply:
@jdpleiness had a call with @marcleblanc2 and discussed this, things look okay and yea grafana shouldn't deploy these unless the specific env vars are set. They're only set by default via our helpers in `frontend` and `migrator`